ADAL/src/ui/mac/ADAuthenticationViewController.m

```diff
@@ -67,6 +68,8 @@ - (BOOL)loadView:(ADAuthenticationError * __autoreleasing *)error
     
     if (_webView)
     {
+        _embeddedWebview = YES;
```

Comment:
LoadView is called once and it shouldn't be called multiple times. However, it’s true, if it gets called multiple times, it will be a problem, but it won’t be fixed by the suggested method (if we don’t have an embedded webview and loadView gets called multiple times, _webView will be set and there will be a different problem). 

So, I moved that logic to a better place :)